test/test_ssl_service.py

```diff
@@ -5,6 +5,11 @@
 
 from conjur.ssl_service import SSLService
 
+"""
+*************** DEVELOPER NOTE ***************
```

Comment:
Not relevant for this PR.
Also, I meant to add message to the assert itself, so when it fails it will be obvious why.

Reply:
as in add the comment above the assert like so:
```
        # These certificate will expire in 2022 which means this test will fail.
        # If this is the case, update the expected output
        self.assertEquals(fingerprint, GITHUB_FINGERPRINT)
        self.assertEquals(readable_certificate, GITHUB_CERT)
```